core/src/main/scala/org/ensime/core/Project.scala

```diff
@@ -26,10 +25,10 @@ final case class ShutdownRequest(reason: String, isError: Boolean = false)
 class Project(
     broadcaster: ActorRef,
     implicit val config: EnsimeConfig
-) extends Actor with ActorLogging with Stash {
+) extends Actor with ActorLogging {
   import context.{ dispatcher, system }
 
-  import FileUtils._
+  import org.ensime.util.FileUtils._
```

Comment:
It was their already - I made it fully qualified rather than having FileUtils imported above and then _ imported here.